src/content/en/2020/css.md

```diff
@@ -464,35 +505,37 @@ Most calculations are very simple, with 99.5% of calculations involving up to 2
+The last global keyword, `unset`, is essentially a hybrid of `initial` and `inherit`. On inherited properties it becomes `inherit` and on the rest it becomes `initial`, essentially resetting the property across all cascade origins. Similarly, to `initial`, it was [defined in 2013](https://www.w3.org/TR/2013/WD-css-cascade-3-20130730/#inherit-initial) and gained [full browser support in 2015](https://caniuse.com/css-unset-value). Despite `unset`'s higher utility, it is used in only 43% of pages, whereas `initial` is used in 51% of pages. Furthermore, besides `max-width` and `min-width`, in every other property `initial` usage outweighs `unset` usage.
 
 {{ figure_markup(
   image="keyword-totals.png",
   caption="Adoption of global keywords as a percent of pages.",
-  description="Bar chart showing the adoption of global keywords as a percent of pages. Mobile pages tend to have a higher adoption rate. The inherit keyword is used on 85% of mobile pages, initial on 51%, unset on 43%, and revert on approximately 0%.",
+  description="Bar chart showing the adoption of global keywords as a percent of pages. Mobile pages tend to have a higher adoption rate. The `inherit` keyword is used on 85% of mobile pages, `initial` on 51%, `unset` on 43%, and `revert` on approximately 0%.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRpe_HsNGpekn6YZV9k6QGmcZPxalqnDrL7DrDY-7X65RZEf_-aGfWuEvhk-yWV83ctIceE1bppCLpj/pubchart?oid=1246886332&format=interactive",
   sheets_gid="437371205",
   sql_file="keyword_totals.sql"
 ) }}
 
 The `all` property was [introduced in 2013](https://www.w3.org/TR/2013/WD-css3-cascade-20130103/#all-shorthand) and gained [near-universal support in 2016 (except Edge) and universal support earlier this year](https://caniuse.com/css-all). It is a shorthand of nearly every property in CSS (except custom properties, `direction`, and `unicode-bidi`), and only accepts the four global keywords as values. It was envisioned as a one liner CSS reset, either as `all: unset` or `all: revert`, depending on what kind of reset we wanted. However, adoption is still very low: we only found `all` on 477 pages (0.01% of all pages), and only used with the `revert` keyword.
 
+{# TODO authors—what are the four global keywords and should we link them here? The spec only lists `initial`, `inherit` and `unset` but then you talk about `revert` so is that the missing 4th one? Should we explain this discrepency? #}
+
```

Comment:
@LeaVerou, @svgeesus, @rachelandrew - thoughts on this?

Reply:
The original three are `initial`, `inherit` and `unset` [defined in CSS values](https://drafts.csswg.org/css-values-4/#common-keywords), and now also  `revert` [defined in CSS Cascade](https://drafts.csswg.org/css-cascade-4/#valdef-all-revert)

Reply:
What about this then:

```suggestion
The `all` property was [introduced in 2013](https://www.w3.org/TR/2013/WD-css3-cascade-20130103/#all-shorthand) and gained [near-universal support in 2016 (except Edge) and universal support earlier this year](https://caniuse.com/css-all). It is a shorthand of nearly every property in CSS (except custom properties, `direction`, and `unicode-bidi`), and only accepts the four global keywords (`initial`, `inherit`, `unset`, and `revert`) as values. It was envisioned as a one liner CSS reset, either as `all: unset` or `all: revert`, depending on what kind of reset we wanted. However, adoption is still very low: we only found `all` on 477 pages (0.01% of all pages), and only used with the `revert` keyword.

{# TODO authors—what are the four global keywords and should we link them here? The spec only lists `initial`, `inherit` and `unset` but then you talk about `revert` so is that the missing 4th one? Should we explain this discrepency? #}

```

Reply:
In case it's not obvious from above with GitHub diff being less than helpful, I just add the bit in parenthesis and removed the TODO - everything else stayed the same:

> ...only accepts the four global keywords (`initial`, `inherit`, `unset`, and `revert`) as values.

Reply:
Or alternatively just hyperlinking [four global keywords](https://drafts.csswg.org/css-cascade-4/#defaulting-keywords) text with that link?

```suggestion
The `all` property was [introduced in 2013](https://www.w3.org/TR/2013/WD-css3-cascade-20130103/#all-shorthand) and gained [near-universal support in 2016 (except Edge) and universal support earlier this year](https://caniuse.com/css-all). It is a shorthand of nearly every property in CSS (except custom properties, `direction`, and `unicode-bidi`), and only accepts the [four global keywords](https://drafts.csswg.org/css-cascade-4/#defaulting-keywords) as values. It was envisioned as a one liner CSS reset, either as `all: unset` or `all: revert`, depending on what kind of reset we wanted. However, adoption is still very low: we only found `all` on 477 pages (0.01% of all pages), and only used with the `revert` keyword.
```

Reply:
Or why not both!:

```suggestion
The `all` property was [introduced in 2013](https://www.w3.org/TR/2013/WD-css3-cascade-20130103/#all-shorthand) and gained [near-universal support in 2016 (except Edge) and universal support earlier this year](https://caniuse.com/css-all). It is a shorthand of nearly every property in CSS (except custom properties, `direction`, and `unicode-bidi`), and only accepts the [four global keywords](https://drafts.csswg.org/css-cascade-4/#defaulting-keywords) (`initial`, `inherit`, `unset`, and `revert`) as values. It was envisioned as a one liner CSS reset, either as `all: unset` or `all: revert`, depending on what kind of reset we wanted. However, adoption is still very low: we only found `all` on 477 pages (0.01% of all pages), and only used with the `revert` keyword.
```